README.md

```diff
@@ -1,117 +1,108 @@
 # OpenZeppelin Contracts for Cairo
 
-[![Tests and linter](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml)
-[![codecov](https://codecov.io/github/OpenZeppelin/cairo-contracts/branch/main/graph/badge.svg?token=LFSZH8RPOL)](https://codecov.io/github/OpenZeppelin/cairo-contracts)
+[![Lint and test](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml)
 
-**A library for secure smart contract development** written in Cairo for [StarkNet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
+**A library for secure smart contract development** written in Cairo for [Starknet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
-## Usage
-
-> ## ⚠️ WARNING! ⚠️
->
+> **Warning**
 > This repo contains highly experimental code.
-> Expect rapid iteration.
+> It has no code coverage checks.
+> It hasn't been audited.
 > **Use at your own risk.**
 
-### First time?
+## Usage
 
-Before installing Cairo on your machine, you need to install `gmp`:
+> **Warning**
```

Comment:
I think both warnings can be merged into one. I wouldn't say I like having two warning blocks so close to each other as the README's presentation.

Reply:
tried it, didn't like much how it looked. each individual message gets lost in a bulky block

Reply:
this is how i see it. the problem with the version you shared is that, although visually "cleaner", it's easier to skip. it's not about how good it looks, but how well it communicates

![image](https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/ba600a5a-60bf-43e2-bbbf-cde2a6e0d064)

Reply:
in the same line, i'd expect the warning from below to disappear before the one from above.